ci-operator/step-registry/ipi/OWNERS

```diff
@@ -0,0 +1,7 @@
+approvers:
```

Comment:
I think we might want to shove someone else into these -- Could we use the author list from the template, please?

```
$ git log --no-merges --pretty=%ae -- ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml | sort | uniq -c | sort -nr | head -n 10
     49 ccoleman@redhat.com
     40 wking@tremily.us
     10 skuznets@redhat.com
      8 vrutkovs@redhat.com
      8 abhinav.dahiya@redhat.com
      7 deads@redhat.com
      7 crawford@redhat.com
      6 ewolinet@redhat.com
      5 cewong@redhat.com
      4 staebler@redhat.com
```